### PR TITLE
add v6 download

### PR DESCRIPTION
**Pull request by jkrauska**
Quick mod to get a v6 table. Addresses #18. I happened to be wanting the same thing, so I was glad to see some thought went in to this.  This is only a slight mode, but works.

### CODE REVIEW

File: pyasn-utils/pyasn_util_download.py [hunk 1]
```diff
@@ -36,20 +36,22 @@
 else:
     from urllib.request import urlopen
 
-if not (len(argv) == 2 and argv[1] == '--latest') and not (len(argv) == 3 and argv[1] == '--dates-from-file'):
+if not (len(argv) == 2 and (argv[1] == '--latest' or argv[1] == '--latestv6')) and not (len(argv) == 3 and argv[1] == '--dates-from-file'):
     print('usage: %s [--dates-from-file FILEWITHDATES] | [--latest]' % (argv[0]))
```
Comment (sebix) [inline pyasn-utils/pyasn_util_download.py]: You need to add `--latestv6` here too.

Comment (jkrauska) [inline pyasn-utils/pyasn_util_download.py]: added help string (which I think is what you were referencing.

**Comment by hadiasghari** [comment]
Thank you, will do a merge asap.

**Comment by hadiasghari** [comment]
Done.

**Pull request merged**